R/derive_vars_dtm.R

```diff
@@ -58,7 +58,7 @@
 #'
 #' mhdt <- tribble(
 #'   ~MHSTDTC,
-#'   "2019-07-18T15:25:40",
+#'   "2019-07-18T15:25",
```

Comment:
The first and the second record are the same. One of them should be removed.

Further updates of the examples are required as the first and third example are the same.
- The third example should be updated such that it shows the behaviour when `ignore_seconds_flag = FALSE` is used.
- An example should be added to show that an error is issued if seconds are present in the DTC variable and `ignore_seconds_flag` is not set to `FALSE`. This should also be mentioned in the description of the `ignore_seconds_flag` argument.

Reply:
@bundfussr 
Working on the changes ... thanks 

> An example should be added to show that an error is issued if seconds are present in the DTC variable and >ignore_seconds_flag is not set to FALSE. 

Meaning:   "2019-07-18T15:25"  will throw error  when ignore_seconds_flag is DEFAULT (ie TRUE)?

Reply:
> Meaning: "2019-07-18T15:25:40" will throw error when ignore_seconds_flag is DEFAULT (ie TRUE)?

Yes